canonical-web-and-design/ubuntu.com
# Update /uasd, /ceph, /pricing/infra

## Done

- Update Ubuntu Advantage service description (/uasd), /ceph, /pricing/infra

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/uasd
    - http://0.0.0.0:8001/ceph
    - http://0.0.0.0:8001/pricing/infra
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to and resolve comments in the copy docs: [/uasd](https://docs.google.com/document/d/1vQNdVmpsB3iQP7SnJHxT3A8rbD5jSAzTEoYUOepR-SM/edit#), [/ceph](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit?ts=5fc77c00#heading=h.zfsh8iinf6jd), [/pricing/infra](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit?ts=5fc77c00#heading=h.zfsh8iinf6jd)


## Issue / Card

Fixes #8359
